supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -20,8 +21,20 @@ Do not rely solely on a shell prompt in a sample command to indicate the require
 If you include a shell prompt to indicate that a user with root privileges must run the command, also include a statement about this requirement in the step text, the introductory text, or the prerequisites.
 ====
 +
-* If a command requires a temporary switch to root privileges, use `sudo` at the beginning of the sample command syntax. If the sample command includes a shell prompt together with `sudo`, the prompt should remain `$` and not be changed to `#`.
-* If multiple commands in a procedure require root privileges, add introductory content to tell the user what to do. For example, "The commands in this procedure require root privileges to run. Either run `su -` to switch to the root user or preface the commands with `sudo`."
+* If a sample command includes a shell prompt together with `sudo`, do not change the prompt to `#`. Keep the `$` prompt, as shown in the following example: 
++
+[source,terminal]
+----
+$ sudo systemctl start firewalld
+----
++
+* If multiple commands in a procedure require root privileges, add introductory content to tell the user about the requirement. 
+The following example shows one way that a writer might integrate a requirement for root access into introductory text for a procedure:
```

Comment:
```suggestion
Here is an example of how to integrate a root access requirement into the introduction for a procedure:
```

Reply:
Thanks. I try to avoid starting sentences with "here is", "there is", or "there are" because these phrases get through the subject and verb of the sentence without really saying very much. "root access requirement" is also too noun-heavy for my tastes.

But I've edited other parts of the sentence to shorten and make more "you"-centric.